### PR TITLE
AB#5536  - Updated documentation with version substitution

### PR DESCRIPTION
**Pull request by russellseymour**
#### 📲 What

Updated documentation generation so that the version number of the CLI to download is set correctly

#### 🤔 Why
		
The documentation is generated each time the software is built. It was originally intended that the docs would be versioned as the binary so that all code snippets would be correct.

A bug in the generation prevented this from happening so that each of the links had a default version number in it.
		
#### 🛠 How
		
Updated the Independent Runner version being used so that the `BUILDNUMBER` environment variable is passed to the container and then parsed as an attribute to AsciiDoc

the `docs.json` file has been updated to include the build number in the title of the generated documentation.

#### 👀 Evidence
		
When the pipeline is run the documentation is now correctly labelled and the snippets have the correct version number.

image

image

		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: docs.json [hunk 1]
```diff
@@ -19,7 +20,8 @@
             "allow-uri-read",
             "toc=left",
             "java=/usr/bin/java",
-            "graphvizdot=/usr/bin/dot"
+            "graphvizdot=/usr/bin/dot",
+            "stackscli_version={{ BUILDNUMBER }}"
         ]
     }
 }
```
Comment (ElvenSpellmaker) [inline docs.json:27]: Is this file ours, if so we should add a newline

File: docs/troubleshooting/access_denied.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+=== Access Denied when running stacks-cli
+
+After the command has been downloaded on a Linux or MacOS computer, the execution bit must be set. If this is not done an Access Denied error will be displayed:
+
+.Access denied running the stacks-cli command
+image::images/stackscli-permission-denied.png[]
+
+The command to set te execution bit is shown below.
```
Comment (ElvenSpellmaker) [inline docs/troubleshooting/access_denied.adoc:8]: `te` --> `the`

File: docs.json (hunk 1 above)
Comment (russellseymour) [reply docs.json]: Fixed in latest push

File: docs/troubleshooting/access_denied.adoc (hunk 2 above)
Comment (russellseymour) [reply docs/troubleshooting/access_denied.adoc]: Fixed in latest push

**Review by ElvenSpellmaker — APPROVED** [review]

**Pull request merged**